src/app/ApplicationTemplate.Shared.Views/Content/Diagnostics/NavigationDebuggerView.xaml

```diff
@@ -92,20 +92,28 @@
 				   Style="{StaticResource NavigationDebuggerTextBlockStyle}" />
 
 		<!-- Clear and Reinitialize buttons -->
-		<StackPanel Orientation="Horizontal"
+		<StackPanel Orientation="Vertical"
 					Spacing="8">
-			
-			<Button Content="Clear all pages"
-					Style="{StaticResource NavigationDebuggerButtonStyle}"
-					Command="{Binding Clear}" />
-
-			<Button Content="Reinitialize navigation"
-					Style="{StaticResource NavigationDebuggerButtonStyle}"
-					Command="{Binding Reinitialize}" />
+			<StackPanel Orientation="Horizontal"
+						Spacing="8">
+				<Button Content="Clear all pages"
+						Style="{StaticResource NavigationDebuggerButtonStyle}"
+						Command="{Binding Clear}" />
 
-			<Button Content="Force update"
-					Style="{StaticResource NavigationDebuggerButtonStyle}"
-					Command="{Binding NavigateToForceUpdatePage}" />
+				<Button Content="Reinitialize navigation"
+						Style="{StaticResource NavigationDebuggerButtonStyle}"
+						Command="{Binding Reinitialize}" />
+			</StackPanel>
+			
+			<StackPanel Orientation="Horizontal"
+						Spacing="8">
+				<Button Content="Force update"
+						Style="{StaticResource NavigationDebuggerButtonStyle}"
+						Command="{Binding NavigateToForceUpdatePage}" />
+				<Button Content="Trigger kill switch"
```

Comment:
That should probably be Toggle instead of Trigger so that you can switch back and forth.